target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-dlan2-2400-ac.dts

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: GPL-2.0 OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "devolo dlan2-2400-ac";
+	compatible = "devolo,dlan2-2400-ac";
```

Comment:
I'd prefer the marketing name here, Devolo Magic 2 WiFi next, as you are using it in the commit title.
devolo,magic-2-wifi-next

We did it similar for the non-next:
https://github.com/openwrt/openwrt/commit/b2130b9ce841d9da2bf5a3d91191192d0c2ad44a

Please use the model property to provide a (more) friendly name.

Reply:
Done